development/tools/jobs/kyma/docsbuilder_test.go

```diff
@@ -17,7 +17,7 @@ func TestDocsBuilderJobPresubmit(t *testing.T) {
 	assert.Len(t, jobConfig.Presubmits, 1)
 	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
 	assert.True(t, ex)
-	assert.Len(t, kymaPresubmits, 1)
```

Comment:
In config.yaml we have defined required_status_checks for release 0.7, IMO we should remove pre-rel07-kyma-gke-upgrade because it is failing right now

Reply:
done